scripts/extension-upload-single.sh

```diff
@@ -33,21 +33,7 @@ script_dir="$(dirname "$(readlink -f "$0")")"
 # calculate SHA256 hash of extension binary
 cat $ext > $ext.append
 
-if [[ $4 == wasm* ]]; then
-  # 0 for custom section
-  # 113 in hex = 275 in decimal, total lenght of what follows (1 + 16 + 2 + 256)
-  # [1(continuation) + 0010011(payload) = \x93, 0(continuation) + 10(payload) = \x02]
-  echo -n -e '\x00' >> $ext.append
-  echo -n -e '\x93\x02' >> $ext.append
-  # 10 in hex = 16 in decimal, lenght of name, 1 byte
-  echo -n -e '\x10' >> $ext.append
-  echo -n -e 'duckdb_signature' >> $ext.append
-  # the name of the WebAssembly custom section, 16 bytes
-  # 100 in hex, 256 in decimal
-  # [1(continuation) + 0000000(payload) = ff, 0(continuation) + 10(payload)],
-  # for a grand total of 2 bytes
-  echo -n -e '\x80\x02' >> $ext.append
-fi
+(( command -v truncate && truncate -s -256 $ext.append )) || ((command -v gtruncate && gtruncate -s -256 $ext.append )) || exit 1
```

Comment:
I'm getting a syntax error here:
```
/Users/sam/Development/duckdb/scripts/extension-upload-single.sh: line 36: ((: command -v truncate && truncate -s -256 build/release/extension/autocomplete/autocomplete.duckdb_extension.append : syntax error in expression (error token is "truncate && truncate -s -256 build/release/extension/autocomplete/autocomplete.duckdb_extension.append ")
```

Reply:
Thanks for double checking this, resolved.